ui/components/multichain/asset-picker-amount/asset-picker-modal/index.scss

```diff
@@ -0,0 +1,71 @@
+.asset-picker-modal {
```

Comment:
This file has ~3 sort of "sibling" selectors.  Could all of this go under `.asset-picker-modal`?  Otherwise we run the risk of affecting other components with CSS from other files.

Reply:
done :)